package.json

```diff
@@ -29,30 +29,30 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "^2.7.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "ember-ajax": "^2.0.1",
+    "ember-cli": "2.7.0",
```

Comment:
My point is if a PR is going to be made against this repo and the repo doesn't work with the latest `ember-cli`, I want the repo fixed to work with the latest before any other changes come in (in case the incoming changes don't play nice with the latest `ember-cli`). At all times I WANT this project working with the latest `ember-cli`, as soon as it isn't that is top priority for this project.